src/types.ts

```diff
@@ -205,3 +206,15 @@ export enum UserIdleState {
   AWAY = "away",
   IDLE = "idle",
 }
+
+export type UIOptions = {
+  canvasActions: CanvasActions;
+};
+
+export type CanvasActions = {
```

Comment:
do we need to export this ? Also this should go before `UIOptions` as readability wise it will be better top to bottom